core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/rendering/RenderingTests.kt

```diff
@@ -196,4 +202,29 @@ class RenderingTests : TestBase() {
         val rendered = schema.toString()
         rendered shouldBe "a: Int?\nb: IntArray\nc: Array<Int>\nd: Array<Int?>"
     }
+
+    @Test
+    fun `render nested FormattedFrame as DataFrame`() {
+        val empty = object : CellAttributes {
+            override fun attributes(): List<Pair<String, String>> = emptyList()
+        }
+        val df = dataFrameOf("b")(1)
+
+        val formatted = dataFrameOf("a")(df.format { all() }.with { empty })
+        val nestedFrame = dataFrameOf("a")(df)
+        val configuration = DisplayConfiguration(enableFallbackStaticTables = false)
+        tableInSessionId = 0
+        val formattedHtml = formatted.toStandaloneHTML(configuration).toString()
+        tableInSessionId = 0
+        val regularHtml = nestedFrame.toStandaloneHTML(configuration).toString()
+
+        formattedHtml.replace("api.FormattedFrame", "DataFrame") shouldBe regularHtml
```

Comment:
maybe also check whether "api.FormattedFrame" is actually present, otherwise this test would succeed if formatting did nothing.

Reply:
Second test checks actual formatting. First test is to emphasize that FormattedFrame should be treated same as DataFrame in toHTML